docs/athletes.md

```diff
@@ -1 +1,84 @@
-Athletes
\ No newline at end of file
+## **¿Qué es Shenanigan?** 🤸‍♀️
+
+
+Shenanigan es una plataforma de código abierto creada para atletas y por atletas. Estamos creando un mundo donde los atletas puedan recibir recompensas directas por superar sus propios límites. Shenanigan quiere brindarles a los atletas una plataforma para crecer, así como construir a su alrededor una comunidad que valore sus esfuerzos.
+
+
+## **¿Porque lo hace?** 🏆
+
+
+**Atleta** = del griego ‘’el que compite con esfuerzo por un premio’’
+
+
+Ahora que conoces la definición de atleta,  interiorízala, y pregúntate, ¿Por qué muchos atletas no tienen la oportunidad de conseguir ese premio ? ¿las oportunidades no son las mismas para todos? ¿solo algunos tienen el talento? ¿porque para algunos el esfuerzo no es proporcional a su premio? Shenanigan plantea ir más allá, la verdadera posibilidad de que cada atleta se ponga en marcha y compita por su premio. Shenanigan entrega a cada atleta los instrumentos web3 necesarios para lograr esto, ¡es tu momento! ¡es tu carrera!
+
+
+## **¿Cómo lo hace?** 📱
+
+
+Nuestros atletas descargan She_Dapp – interactúan con la interfaz y completan la información relevante para iniciar con un desafío.
+
+Ejemplo:
+
+•	**¿Que deporte practicas?**
+
+Atletismo
+
+•	**¿Qué categoría específica del deporte?**
+
+Media distancia - 1500m
+
+•	**Agrega un título para tu desafío**
+
+Correr 1500m en menos de 5 minutos
+
+•	**Describe el desafío que estás a punto de intentar**
+
+Quiero correr 1500m en menos de 5 minutos
+
+Después de completar esta información que será la data inicial de su desafío, Shenanigan acuña la NRG Card con su respectivo FAN Token, el FAN CLUB, y guarda el desafío para acuñar la CHALLENGE Card.
```

Comment:
These are athletes. They won't know what this means. We should tell them about the power of personal tokens, NFTs, and web3 markets. For example. 

"We will mint you a signed trading card..."